benches/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "benches"
```

Comment:
Curious about the motivation to create a new `benches` crate, rather than putting the cache benchmarks in `linkerd/cache/benches`? Not a blocker, I just want to know if there's a reason we can't do the former.

Reply:
All other things equal, I think I'd prefer that the tests are "close" to the code they test.. I.e. rather than a crate for all benches, I think they should be within the crate they test.  My workflow is to run `cargo test -p $crate`.

Reply:
No particular reason really. I was looking at other crates on github and how they organize things and its was mostly 50/50. Makes sense to be closer to code. I moved it and switched to libtest. Feel free to take another look. I am not sure whether the `unstable` feature flag is how it should be setup.

Reply:
>  I was looking at other crates on github and how they organize things and its was mostly 50/50

In general, I think putting benchmarks in a separate crate is something that libraries will do, usually so that projects that depend on that library don't need to download test-only dependencies. In this case, I don't think it's necessary, since there aren't any downstream consumers of linkerd-internal crates.